ntfs_fr.md

```diff
@@ -528,6 +529,48 @@ address_id | chaine | Requis | Identifiant de l'adresse
 street_name | chaine | Requis | Nom de la voierie
 house_number | chaine | Optionnel | Numéro du seuil
 
+### occupancies.txt (optionnel)
+Pour chaque ligne de ce fichier, l'affluence est appliquée à toutes les séries d'horaires successifs d'une circulation remplissant l'ensemble des conditions suivantes :
+
+* les poteaux d'arrêts des horaires se trouvent entre `from_stop_area` et `to_stop_area` (inclus)
+* le jour de circulation est inclus entre `from_date` et `to_date`
+* le jour de semaine de la circulation est actif (voir champs `monday`, `tuesday`, etc.)
+* les `departure_time` des horaires sont inclus entre `from_time` et `to_time` (inclus)
+* il est possible de monter dans le véhicule sur le premier des horaires
+* il est possible de descendre dans le véhicule sur le dernier des horaires
```

Comment:
Proposition qui me semble plus claire (qui au moins reformule ce que j'ai compris) :
```suggestion
La notion d'affluence est appliquée sur l'intervalle entre 2 horaires (affluence à bord du véhicule).
Pour chaque ligne de ce fichier, l'affluence est appliquée à tous les intervalles d'une circulation entre deux horaires (successifs ou non) remplissant l'ensemble des conditions suivantes :

* le premier des intervalles débute à l'arrêt `from_stop_area` et le dernier des intervalles se termine l'arrêt `to_stop_area` (inclus)
* le jour de circulation est inclus entre `from_date` et `to_date` (inclus)
* le jour de semaine de la circulation est actif (voir champs `monday`, `tuesday`, etc.)
* les `departure_time` des horaires sont inclus entre `from_time` et `to_time` (inclus)
* il est possible de monter dans le véhicule à l'arrêt `from_stop_area`
* il est possible de descendre du véhicule à l'arrêt `to_stop_area`
```

Reply:
Nous avons eu une longue discussion avec @woshilapin sur le sujet. J'aime bien ta proposition de reformulation @pbougue mais si cette proposition fiabilise l'info pour éviter une information parfois fausse en cas de point de passage, on va se heurter à un soucis de prédictabilité de l'info sur des enrichissements côté Data.
J'ai proposé de retirer les critères de MI/DI pour appliquer systématiquement la règle entre le `from_stom_area` et le `to_stop_area`, mais les effets de bords  sont-ils acceptables ?
A suivre :christmas_tree: :santa:

Reply:
Correction of the text with your proposition done in 99c9dd7.

Reply:
Je vous laisse voir s'il faut relancer la discussion de fiabilité vs. MDI (autre thread car ne concerne qu'une sous-partie de ce texte ?).
Je laisse donc ce thread ouvert, mais pour ce qui me concerne, c'est OK de le 'resolve'.

Reply:
J'ai assez envie de commencer l'implémentation pour avoir du _feedback_ concret à ce stade.

Reply:
Je pense que cette version est good enough